mkdocs.yml

```
@@ -75,14 +75,15 @@ nav:
         - geps/gep-1324.md
         - geps/gep-1282.md
       - Implementable:
-        - geps/gep-1364.md
+        - DummyEntry
```

I think this was a temporary change for the sake of CI?
```suggestion
        - DummyEntry
```

---

no, `mkdocs` will permanently crash if the object is empty. Actually, let me see if I can add the empty list instead...

---

Maybe just remove "Implementable" altogether until we have a GEP that moves to that status?